### PR TITLE
[IMP] core: rename upgrades-paths to upgrade_path

### PR DESCRIPTION
**Pull request by Xavier-Do**

### CODE REVIEW

File: odoo/modules/migration.py [hunk 1]
```diff
@@ -85,8 +85,8 @@ def get_scripts(path):
 
             self.migrations[pkg.name] = {
                 'module': get_scripts(get_resource_path(pkg.name, 'migrations')),
-                'module_upgrades': get_scripts(get_resource_path(pkg.name, 'upgrades')),
-                'upgrades': get_scripts(_get_upgrades_paths(pkg.name)),
+                'module_upgrades': get_scripts(get_resource_path(pkg.name, 'upgrade')),
```
Comment (KangOl) [inline odoo/modules/migration.py]: I'm not sure about this change.
I think `upgrades` still make sense for the directory inside the modules. It's the directory you put your `upgrades` in.

File: odoo/modules/migration.py [hunk 2]
```diff
@@ -130,12 +130,12 @@ def _get_migration_files(pkg, version, stage):
 
             mapping = {
                 'module': opj(pkg.name, 'migrations'),
-                'module_upgrades': opj(pkg.name, 'upgrades'),
+                'module_upgrades': opj(pkg.name, 'upgrade'),
```
Comment (KangOl) [inline odoo/modules/migration.py]: Ditto.

File: odoo/modules/migration.py (hunk 1 above)
Comment (KangOl) [reply odoo/modules/migration.py]: It's should also be in par with the legacy `migrations` directory.

Comment (Xavier-Do) [reply odoo/modules/migration.py]: Is it already in use?

Comment (KangOl) [reply odoo/modules/migration.py]: maybe...

**Comment by KangOl** [comment]
@robodoo r+

**Pull request closed without merging**